contents/product-engineers/llm-product-metrics.md

```diff
@@ -0,0 +1,157 @@
+---
+date: 2024-05-03
+title: 'Product metrics to track for LLM apps'
+author:
+  - lior-neu-ner
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/posthog-marketing/marketing-hog.jpg
+featuredImageType: full
+category: Engineering
+tags:
+  - Guides
+  - Product metrics
+crosspost:
+  - Blog
+---
+
+import { ProductScreenshot } from 'components/ProductScreenshot'
+
+LLM-powered apps differ in three key ways when compared to regular SaaS apps:
+
+1. Their costs increase as as app usage increases.
+2. Request latency spikes and errors are more common.
+3. You're not always sure if your LLM output is what was expected.
```

Comment:
Could you use 1 subject here instead of 3 like:
```suggestion
1. Costs increase as as app usage increases.
2. Request latency spikes and errors are more common.
3. LLM output is unpredictable and can lead to a negative user experience.
```